tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_wifi.py

```diff
@@ -20,7 +20,9 @@ def test_connect_to_wifi_via_settings_app(self):
         settings.launch()
         wifi_settings = settings.open_wifi()
 
+        self.assertFalse(wifi_settings.is_wifi_enabled, "WiFi should be disabled")
```

Comment:
Sorry, I only saw this comment until I checked in.
Why you say this assert doesn't add much value? If wifi is disabled, but in the settings it shows as enabled, then this would catch it.